deployment/v3/mosip/mock-mv/install.sh

```diff
@@ -22,7 +22,7 @@ function installing_mockmv() {
   helm repo update
 
   echo Installing mock-mv
-  helm -n $NS install mock-mv mosip/mock-mv --version $CHART_VERSION
+  helm -n $NS install mock-mv mosip/mock-mv --set image.repository=mosipdev/mock-mv --set image.tag=develop --version $CHART_VERSION
```

Comment:
by defaut the tag for images are develop in 12.0.2 version of charts.
So setting tag can be ignored.